polyfill/lib/ecmascript.mjs

```diff
@@ -60,7 +60,6 @@ import * as PARSE from './regex.mjs';
 const ES2019 = {
   Call,
   SpeciesConstructor,
-  ToInteger,
```

Comment:
the actual ES object in a real polyfill will always have ToInteger on it; this doesn't seem particularly valuable.